### ISSUE TITLE
For Cycle 3

### ISSUE THREAD

**Issue by HarryMacfinned** [bisq-network/compensation#308]
## Summary

> _Specify the total amount of BSQ you are requesting_

 - BSQ requested: `2900`

This compensation request :
- follows the https://docs.bisq.network/roles.html#compensation recommandations
- applies the 1BSQ=1USD rule (as still suggested on the slack #compensation channel)

### Substantive reports

- Support | https://github.com/bisq-network/roles/issues/64#issuecomment-509329039 | (BSQ requested = `2500`)
Evaluation is based and intended in line with : previous compensation requests and other support compensation requests.

- Fees and connectivity monitoring | see the weekly reports in https://github.com/bisq-network/roles/issues/10 | (BSQ requested = `200`)
Concerned monitored data files : https://github.com/HarryMacfinned/bisq-monitor
TL;DR: NTR, nodes were delivering coherent fees.

- Prices monitoring | https://github.com/bisq-network/roles/issues/10#issuecomment-509290795 | (BSQ requested = `200`)
TL;DR: 4 issues this cycle (all rapidly corrected)

NB1 : The monitoring role includes also manual sorting of false alarms, ie monitoring several Bisq specific pages + following of Tor metrics page + maillist
NB2 : All the alerts are given on the slack #bisq-monitor and #bisq-pricenode channels

## Others
- timestamp code added to the monitoring code. Operational (and delivered 1st alerts) [`0 BSQ`]
- Clearnet Bisq webpages monitoring : reported in the #bisq-website channel [`0 BSQ`]

**Comment by ManfredKarrer** [comment]
Your link in the app is wrong:
https://bisq-network/dao-compensation/308
-> https://bisq.network/dao-compensation/308

**Comment by HarryMacfinned** [comment]
ah, thanks @ManfredKarrer
I guess the only way to modify is to remove this comp. request in the DAO and resubmit.

**Comment by ManfredKarrer** [comment]
I think its not needed to resubmit IMO...
